### PR TITLE
fix(microbenchmarking): pass es index as cli parameter

### PR DESCRIPTION
**Pull request by aleksbykov**
After changing es index for microbenchmarking test resutls, unit tests start failing. Unit test try to get response from file and if not it try to get data from ES. But after changing ES index return results are empty.

Add new cli parameter for microbenchmarking to pass ES index name to work with.

unit tests are passed

Fixes: #6144

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
fix #6144

File: unit_tests/test_microbenchmarking.py [hunk 1]
```diff
@@ -267,11 +269,18 @@ def test_help_parameter_for_check(self):
     def test_execute_mbm_from_cli(self):
         result_path = os.path.join(os.path.dirname(__file__), 'test_data/test_microbenchmarking/PFF_with_AVGAIO')
         html_report = tempfile.mkstemp(suffix=".html", prefix="microbenchmarking-")[1]
+        LOGGER.info(['./microbenchmarking.py', 'check',
```
Comment (fruch) [inline unit_tests/test_microbenchmarking.py:272]: just put the arguments in a list, not much point duplicating it.

File: unit_tests/test_microbenchmarking.py [hunk 2]
```diff
@@ -65,10 +65,12 @@ def _get_hash(*args, **kwargs):
 
 class TestMBM(unittest.TestCase):  # pylint: disable=too-many-public-methods
     def setUp(self):
-        self.mbra = MicroBenchmarkingResultsAnalyzerMock(email_recipients=('alex.bykov@scylladb.com', ))
+        self.mbra = MicroBenchmarkingResultsAnalyzerMock(email_recipients=(
+            'alex.bykov@scylladb.com', ), es_index="microbenchmarking")
         self.mbra.hostname = 'godzilla.cloudius-systems.com'
         self.cwd = os.path.join(os.path.dirname(__file__), '..', 'sdcm')
         self.mbra.test_run_date = "2019-06-27_11:39:40"
+        self.es_index = "microbenchmarking"
```
Comment (fruch) [inline unit_tests/test_microbenchmarking.py:73]: don't we want some coverage that the new index also works ?

Comment (aleksbykov) [reply unit_tests/test_microbenchmarking.py:73]: Once we have more results in it and old microbenchmark will be removed, we switch to new index also unittests. 
Logic was not changed at all how microbenchmark get data, only index name

Comment (fruch) [reply unit_tests/test_microbenchmarking.py:73]: I would recommend open a task for it, not forgetting that unittest should be part of removal of the old index

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**